ui/ducks/send/send.js

```diff
@@ -360,6 +365,29 @@ export const computeEstimatedGasLimit = createAsyncThunk(
   },
 );
 
+/**
+ * This method is used to keep the original logic from the gas.duck.js file
+ * after receiving a gasPrice from eth_gasPrice. First, the returned gasPrice
+ * was converted to GWEI, then it was converted to a Number, then in the send
+ * duck (here) we would use getGasPriceInHexWei to get back to hexWei. Now that
+ * we receive a GWEI estimate from the controller, we still need to do this
+ * weird conversion to get the proper rounding.
+ * @param {T} gasPriceEstimate
+ * @returns
+ */
+function getRoundedGasPrice(gasPriceEstimate) {
```

Comment:
Nit: feels like this should be be in some `util` type file

Reply:
I disagree, as it will only ever be used here I hope, and maybe will eventually be removed entirely. Moving things to Util files is a vote of confidence for their usage and it is not exported on purpose.